internal/rbd/types/volume.go

```diff
@@ -25,6 +25,8 @@ import (
 
 //nolint:interfacebloat // more than 10 methods are needed for the interface
 type Volume interface {
+	// Mirror interface implementation for Volume
+	Mirror
```

Comment:
because you use `mirror = rbdVol`, I would expect that the `Mirror` interface is kept seperate. I do not see a need to have it part of the Volume interface. The `rbdVolume` struct can still implement both interfaces.